pkg/liqo-controller-manager/resource-request-controller/resourceRequest_controller_methods.go

```diff
@@ -129,3 +128,13 @@ func (r *ResourceRequestReconciler) checkOfferState(ctx context.Context,
 
 	return nil
 }
+
+// getOfferName returns the name of the ResourceOffer coming from the given cluster.
+func getOfferName(cluster discoveryv1alpha1.ClusterIdentity) string {
+	return fmt.Sprintf("resourceoffer-%s", cluster.ClusterName)
+}
+
+// getTenantName returns the name of the Tenant for the given cluster.
+func getTenantName(cluster discoveryv1alpha1.ClusterIdentity) string {
+	return fmt.Sprintf("tenant-%s", cluster.ClusterName)
+}
```

Comment:
Same here

Reply:
The UniqueName is not needed for the tenant, but for it to be safe it must be derived from the ClusterIdentity for the local ForeignCluster, not the ClusterIdentity provided in the ResourceRequest. I fixed that, thanks.

I also removed the prefix.